### ISSUE TITLE
Refactor tests to use unittest module/class setup/teardown

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#364]
`unittest` provides hooks to setup and tear down tests on the class and module level, not just the method level.  We should refactor our tests to use these instead of the `global` keyword hackery we're using now.

https://docs.python.org/2/library/unittest.html#setupclass-and-teardownclass

**Comment by dcolligan** [comment]
Tests affected:
```
$ ack global
unit/test_imports.py
30:def globalSetUp():
33:    global graph
47:        self.graph = globalSetUp()

unit/test_protocol_errors.py
22:    global _app
28:    global _app

unit/test_simulated_stack.py
31:    global _app
36:    global _app
46:        global _app

unit/test_views.py
28:    global _app
33:    global _app
44:        global _app
```

**Comment by dcolligan** [comment]
Fixed in #365
